### DISCUSSION TITLE
Max steps for male and female

### DISCUSSION THREAD

**Discussion by ellalewi547** [RangeShifter/RangeshiftR-tutorials#44] [Q&A]
Hello, I wondered if I am able to define different max steps for females and males? Forgive me if there is detail on this and I haven't come across it.

**Comment by JetteReeg** [comment]
Hi Ella,
yes, you can have sex dependent maximal number of steps. There was a bug in the code, why the parameter was not set correctly. I have merged this bugfix into the main branch. You should update your RangeShiftR package with the newest version and then be able to set the MaxSteps parameter sex dependent (like the MinSteps parameter).
Best,
Jette

**Reply by ellalewi547** [reply]
I am currently using version 1.0.4 and have tried to reinstall using `devtools::install_github("https://github.com/RangeShifter/RangeShiftR-pkg", ref = "main", subdir="RangeShiftR")`, do I need to reinstall a developers version instead as the version has not changed?

**Reply by JetteReeg** [reply]
Thanks for hinting to that issue. I forgot to change the version number for the R package. The link you have posted is correct and should install the latest version with the bugfix.

**Comment by ellalewi547** [comment]
Ah perfect, thank you very much!
